frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -264,12 +264,16 @@ export const ListPageCreateDropdown: React.FC<ListPageCreateDropdownProps> = req
  * @param {boolean} loaded - indicates that data has loaded
  * @param {function} onFilterChange - callback function for when filter is updated
  * @param {RowFilter[]} [rowFilters] - (optional) An array of RowFilter elements that define the available filter options
+ * @param {string} [labelFilter] - (optional) a unique name key for label filter
```

Comment:
```suggestion
 * @param {string} [labelFilter] - (optional) a unique key for label filter. This may be useful if there are multiple `ListPageFilter` components rendered at once.
```

Reply:
done. thanks :)